frontside/lib/routers/routers.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:beamer/beamer.dart';
+import 'package:frontside/views/about/about.dart';
+import 'package:frontside/views/home/home.dart';
+
+
+
+class RouteManager{
+  static const String homeRoute = "/";
+  static const String aboutRoute = "/about";
```

Comment:
I would rather move this to view classes to keep things as close as possible. You could name this as `route` or `path`. For example, in `lib/views/home/home.dart`:
```dart
static const String route = "/"
```

and then use it here.